Dockerfile

```diff
@@ -19,9 +19,19 @@
 # submit itself to any jurisdiction.
 
 FROM python:3.5
+
+ENV TERM=xterm
+RUN apt-get update && \
+    apt-get install -y vim emacs-nox
+
 ADD . /code
 WORKDIR /code
-RUN pip install -e .[all]
+
+# Debug off by default
+ARG DEBUG=false
+
+RUN if [ "${DEBUG}" = "true" ]; then pip install wdb; pip install -e .[all]; else pip install .[all]; fi;
```

Comment:
What about using `requirements-dev.txt` so that we could add there `Flask-DebugToolBar` and friends if need be?